usr/src/cmd/zdb/zdb.c

```diff
@@ -3530,14 +3686,14 @@ main(int argc, char **argv)
 	nvlist_t *policy = NULL;
 	uint64_t max_txg = UINT64_MAX;
 	int rewind = ZPOOL_NEVER_REWIND;
+	const char *flags = "bcdehilmop:r:st:uvx:ACDFI:LMPRSU:X";
```

Comment:
I'm going to trust you that this has the same letters as before plus 'o' :-)
Someday we should add long options to this.  Otherwise we're going to run out of letters!